src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -36,10 +40,10 @@ def check_sync(self):
             elif os.stat(path).st_mtime < time.time() - 86400 * 5:
                 unlink = True
             else:
-                has_core_file = True
+                corefiles.append(file)
 
             if unlink:
                 os.unlink(path)
 
-        if has_core_file:
-            return Alert(CoreFilesArePresentAlertClass)
+        if corefiles:
+            return Alert(CoreFilesArePresentAlertClass, {"corefiles": ', '.join(corefiles)})
```

Comment:
@william-gr 's opinion was: "Not sure if there is any usefulness in listing every file here."

https://github.com/truenas/middleware/pull/6120/files/72c308edfeed2f289fae614f6b6a610cada1f60b#diff-af469e38245d7ba3120168e3b197e205fdfcd036e12b79d85af3cbc7a5f18771

Or has anything changed?

Reply:
I honestly wasn't aware that we had gone down this road before. Big picture, I think having the filenames in the alert will help with triaging.

Reply:
Yes, it seems I was wrong and having them may be useful after all